inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -183,15 +183,31 @@ class XmlDeserializer : public ngraph::AttributeVisitor {
     /// \param node xml op representation
     std::vector<std::shared_ptr<ngraph::op::util::SubGraphOp::InputDescription>>
     parseInputDescription(const pugi::xml_node& node);
+    /// \brief Traverses port_map in order to create vector of MultiSubGraphOp::InputDescription shared_ptrs.
+    /// Shall be used only for ops which have attributes similar to port_maps.
+    /// \param node xml op representation
+    /// \param port_map_name port_map attribute name in IR
+    std::vector<std::shared_ptr<ngraph::op::util::MultiSubGraphOp::InputDescription>>
+    parseMSInputDescription(const pugi::xml_node& node, std::string port_map_name);
     /// \brief Traverses port_map in order to create vector of OutputDescription shared_ptrs.
     /// Shall be used only for ops which have port_map attribute.
     /// \param node xml op representation
     std::vector<std::shared_ptr<ngraph::op::util::SubGraphOp::OutputDescription>>
     parseOutputDescription(const pugi::xml_node& node);
+    /// \brief Traverses port_map in order to create vector of MultiSubGraphOp::OutputDescription shared_ptrs.
+    /// Shall be used only for ops which have attributes similar to port_maps.
+    /// \param node xml op representation
+    /// \param port_map_name port_map attribute name in IR
+    std::vector<std::shared_ptr<ngraph::op::util::MultiSubGraphOp::OutputDescription>>
+    parseMSOutputDescription(const pugi::xml_node& node, std::string port_map_name);
```

Comment:
As I see here we also have some specific from MultiSabGraph.
I propose to refactor this code and use the common structures for subgraphs representation and Input/Output descriptions.